keysManagement/interface.go

```diff
@@ -1,17 +1,29 @@
 package keysManagement
 
 import (
+	"time"
+
 	"github.com/multiversx/mx-chain-core-go/core"
 	crypto "github.com/multiversx/mx-chain-crypto-go"
 )
 
 // ManagedPeersHolder defines the operations of an entity that holds managed identities for a node
 type ManagedPeersHolder interface {
```

Comment:
so now we have the interface in the same package as the implementation. Because we need it in several packages.